daemon/main.c

```diff
@@ -400,11 +400,24 @@ static const char *verify_required_directory(const char *dir) {
 #ifdef ENABLE_HTTPS
 static void security_init(){
     char filename[FILENAME_MAX + 1];
-    snprintfz(filename, FILENAME_MAX, "%s/ssl/key.pem",netdata_configured_user_config_dir);
-    security_key    = config_get(CONFIG_SECTION_WEB, "ssl key",  filename);
+    security_key    = config_get(CONFIG_SECTION_WEB, "ssl key",  "none");
+    if (!strcmp(security_key, "none")) {
+        snprintfz(filename, FILENAME_MAX, "%s/ssl/key.pem",netdata_configured_user_config_dir);
+        security_key    = config_get(CONFIG_SECTION_WEB, "tls key",  filename);
+    }
+
+    security_cert    = config_get(CONFIG_SECTION_WEB, "ssl certificate",  "none");
+    if (!strcmp(security_cert, "none")) {
```

Comment:
Why does `"ssl certificate"` take priority over `"tls certificate"`?

Reply:
The same explanation given before. This is during the initialization, so we will test only one time.